### ISSUE TITLE
Illegal Unexpected Token 

### ISSUE THREAD

**Issue by sf-asparks** [tsqllint/tsqllint#33]
When installing node package I receive the following:

OS Windows_NT 6.2.9200
Node 0.10.40
NPM 1.4.28
```
execSync(`call ${process.env.APPDATA}/npm/node_modules/tsqllint/scripts/instal
         ^
SyntaxError: Unexpected token ILLEGAL
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
```

**Comment by dougwilson** [comment]
Hi @sf-asparks this package requires Node.js 4 or higher to install from npm. @nathan-boyd should we add that to the README?

**Comment by sf-asparks** [comment]
I actually looked for that information first but couldn't find it. I would either add it to the read me or add it to the npm package, https://docs.npmjs.com/files/package.json#engines

**Comment by dougwilson** [comment]
I went ahead and put together PR #34 . Feel free to throw up your own PR if you think anything can be helpful!

**Comment by nathan-boyd** [comment]
Resolved in #34
